database_setup.sql

```diff
@@ -1,6 +1,10 @@
 CREATE DATABASE IF NOT EXISTS groot_caffeine_service_dev;
```

Comment:
Yeah I was just following a convention of a `_dev` suffix for development environments and just the service name with no suffix for prod (at least for ruby services).